content/applications/essentials/contacts.rst

```diff
@@ -178,13 +178,64 @@ section, and other partner options, including :guilabel:`Partner Activation` and
 Review` configurations. These are **only** present when the *Enterprise Resellers* module is
 installed.
 
+To publish a list of all active partners on the website, a new webpage must be created first.
+Navigate to :menuselection:`Website app --> Site --> Pages` and click :guilabel:`New`. Hover over
+the :guilabel:`Blank Page` template, and click the :guilabel:`Use this template` button to create a
+new page.
+
+In the :guilabel:`New Page` pop-up window, enter `Partners` in the :guilabel:`Page Title` field.
```

Comment:
```suggestion
In the :guilabel:`New Page` pop-up window that appears, enter an appropriate title, like `Partners`, in the :guilabel:`Page Title` field.
```
just some added specificity.

also, do you want to consider mentioning anything about the 'Add to menu' toggle switch that is present in this pop-up window? It could be beneficial to briefly mention this to the reader, as well. Just a suggestion...